### DISCUSSION TITLE
Can xdrip upload companion app data to Dexcom share?

### DISCUSSION THREAD

**Discussion by williknow** [NightscoutFoundation/xDrip#3684] [Q&A]
Hi helpfull people,

I am trying to upload data from Camapsfx to Dexcom share. The Dexcom G6 Sensor is directly connected to the Camapsfx App. 
Xdrip is receiving the BG Values and is also sharing to xdrip followers. So far, so good.
I would like to upload the data to Dexcom share, as well. 
I have made the settings within xdrip but it does not work. I could not find logs stating any problems, but nothing is uploaded to Dexcom share. 
Is it actually possible to send  data to Dexcom share which is not received via Bluetooth?
Thanks for any experiences or help on this topic.

**Comment by Navid200** [comment]
Considering you have already set up xDrip Sync and have followers, why do you need to upload to Dex share?

**Reply by williknow** [reply]
Because there is an integration for dexcom share in home assistant. The xdrip app has this option and I would like to know how it is supposed to work.
